workshops/lib/twilio-basic/signup.md

```diff
@@ -122,18 +125,19 @@ Leave the "Company/Project URL" and "Phone Number" fields blank
 
 > ![](img/leave_blank.png)
 
-**Entering The Promo Code**
+### Entering The Promo Code
 
 Click "I have a promo code."
 
 > ![](img/i_have_a_promo_code.png)
 
-To get the actual promo code:
+To get the promo code:
+
+  - Club members: Ask your club leader for the Twilio promo code.
 
-- Ask your club leader for the Twilio promo code
-- If you are the club leader, send an email to the email address
-  `jonathanleung1337@gmail.com` with your full name and school and ask for the
-  Twilio promo code. They will respond to you quickly.
+  - Club leaders: send an email to the email address `jonathan@hackclub.io` with
```

Comment:
@jonleung Can the autoresponder only be run on your jonathanleung1337@gmail.com account?